include/CL/sycl/address_space/detail/address_space.hpp

```diff
@@ -143,7 +149,8 @@ struct address_space_base {
 
       \todo Add to the specification
   */
-  using opencl_type = typename opencl_type<T, AS>::type;
+  //using opencl_type = typename opencl_type<T, AS>::type;
+  using opencl_type = typename ocl_type<T, AS>::type;
```

Comment:
Indeed. I have already filed a [bug report](https://connect.microsoft.com/VisualStudio/feedback/details/3113067/type-alias-fails-to-set-while-accessing-member-alias-from-class-with-identical-name) with the name: _type alias fails to set while accessing member alias from class with identical name_. Nomen est omen.